### PR TITLE
Guild Hook Blog Post!

### PR DESCRIPTION
**Pull request by julien51**
# Description

# Issues

Fixes #
Refs #

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [x] I have performed a self-review of my own code
- [x] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by julien51** [comment]
Draft build https://648b80a58c49450e6422e7c8--unlock-protocol-com-staging.netlify.app/blog

File: unlock-protocol-com/blog/guild-hook.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: Guild.xyz hook
+subTitle: Easily create all kinds of allow-list on your lock using Guild.xyz!
+description: By leveraging a new kind of smart contract hook, you can easily create all kinds of allow-list on your lock using Guild.xyz.
+author: Julien Genestoux
+publishDate: June 15, 2023
+image: /images/blog/guild-hook/guild.xyz.jpeg
```
Comment (ccarfi) [inline unlock-protocol-com/blog/guild-hook.md:7]: @julien51 please swap for this image for brand consistency https://unlock-protocol.com/guides/wp-content/uploads/2022/06/guild.xyz_.png

File: unlock-protocol-com/blog/guild-hook.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+1. First, I will go to [Guild.xyz](http://Guild.xyz) and create a Guild whose **requirement** is to follow Unlock on Twitter.
+
+![Guild.xyz Requirements](/images/blog/guild-hook/requirements.png)
+
+2. Then, once the [Guild has been created](https://guild.xyz/the-unlock-followers), I come back to the Unlock dashboard, go to the advanced settings for my lock, and select the Hooks section.
+
+![Guild.xyz Settings](/images/blog/guild-hook/unlock-hook-settings.png)
+
+3. I can then save the changes and my membership contract is now **\*\***hooked**\*\*** to the Guild of the Unlock Protocol followers on Twitter!
```
Comment (ccarfi) [inline unlock-protocol-com/blog/guild-hook.md:32]: @julien51 not sure if this was going for bold in markdown? lots of asterisks

File: unlock-protocol-com/blog/guild-hook.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+Now the requirements are set, and there is one more step to go.
+
+The last step is to build a checkout URL, [like this one](https://app.unlock-protocol.com/checkout?id=23d15cce-8423-4761-ba48-6dd5ab5a048c), through which users are prompted to join the Guild, if they don’t already, before they can mint their NFT membership!
```
Comment (ccarfi) [inline unlock-protocol-com/blog/guild-hook.md:36]:
```suggestion
The last step is to build a checkout URL, [like this one](https://app.unlock-protocol.com/checkout?id=23d15cce-8423-4761-ba48-6dd5ab5a048c), through which users are prompted to join the Guild if they are not already members. Once someone is a member of the Guild, they can mint their NFT membership!
```

**Review by ccarfi — APPROVED** [review]
approved with a few comments and suggestions

File: unlock-protocol-com/blog/guild-hook.md (hunk 1 above)
Comment (julien51) [inline unlock-protocol-com/blog/guild-hook.md:7]:
```suggestion
image: /guides/wp-content/uploads/2022/06/guild.xyz_.png
```

**Pull request merged**